examples/users.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$projectId = '';
+$secretKey = '';
+
+$httpOptions = ['base_uri' => CODEMASH_API_URL];
+
+$httpClient = new \GuzzleHttp\Client($httpOptions);
+
+$cmClient = new \Codemash\CodemashClient($secretKey, $projectId, $httpClient);
+
+$cmUser = new \Codemash\CodemashUser($cmClient);
```

Comment:
I think we need to stick to common naming convention through out all the libs we have currently (C#, Js.) https://github.com/codemash-io/CodeMash.Net/tree/master/src

1. Codemash rename to CodeMash
2. CodemashUser ? ( I think we need rename that to CodeMashMembership in regards to module name. $cmUser->getUsers() doesn't sound to me right.

Reply:
Will make a separate PR, once that gets merged will rebase this one aftewards.

Reply:
@codemash-io related question: do you really want to keep "CodeMash" prefix on all class names?

Currently it would look like this: `\CodeMash\CodeMashClient`, `\CodeMash\CodeMashMembership`, etc.

To my understanding to reduce reading noise it would make more sense to have it like this: `\CodeMash\Client`, `\CodeMash\Membership`, etc.

Reply:
Totally agree about the noise you've mentioned \CodeMash\CodeMashClient We need to get rid of it :)